university-of-new-england-australia-note.csl

```diff
@@ -0,0 +1,558 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>University of New England, Australia (note)</title>
+    <id>http://www.zotero.org/styles/university-of-new-england-australia-note</id>
+    <link href="http://www.zotero.org/styles/university-of-new-england-australia-note" rel="self"/>
+    <link href="http://www.zotero.org/styles/universita-di-bologna-lettere" rel="template"/>
+    <link href="http://www.une.edu.au/__data/assets/pdf_file/0007/87811/REF_Trad.-Footnoting.pdf" rel="documentation"/>
+    <author>
+      <name>Benjamin Hollister</name>
+      <email>ben@rootsandbranches.com.au</email>
+      <uri>http://www.mendeley.com/profiles/benjamin-hollister2/</uri>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <category field="history"/>
+    <summary>University of New England (Australia) Discipline of History Traditional Footnoting</summary>
+    <updated>2015-09-05T21:56:36+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor-translator">
+    <group delimiter=", ">
+      <choose>
+        <if variable="author">
+          <names variable="editor" delimiter=", ">
+            <name delimiter=", "/>
+            <label form="verb" prefix=" (" suffix=") "/>
+          </names>
+          <choose>
+            <if variable="container-author">
+              <group>
+                <text term="in" text-case="lowercase" suffix=" "/>
+                <names variable="container-author">
+                  <name delimiter=", "/>
+                </names>
+              </group>
+            </if>
+          </choose>
+        </if>
+      </choose>
+      <choose>
+        <if variable="author editor" match="any">
+          <names variable="translator" delimiter=", ">
+            <label form="verb" prefix=" (" suffix=") "/>
+            <name delimiter=", "/>
+          </names>
+        </if>
+      </choose>
+    </group>
+  </macro>
+  <macro name="secondary-contributors">
+    <choose>
+      <if type="chapter paper-conference" match="none">
+        <text macro="editor-translator"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="container-contributors">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <text macro="editor-translator"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="editor-note">
+    <names variable="editor">
+      <name sort-separator=", " delimiter=", "/>
+      <label form="verb" prefix=" (" suffix=")" strip-periods="true"/>
+    </names>
+  </macro>
+  <macro name="translator-note">
+    <names variable="translator">
+      <name sort-separator=", " delimiter=", "/>
+      <label form="verb-short" prefix=", "/>
```

Comment:
I guess I don't quite understand why you don't want to fix this? I'm not asking for major revisions here. 
We're shipping these styles out to a userbase that's likely in the millions, so I'm not going to accept a style that I *know* does something wrong to the repository (this is, btw., correct in the template Italian style. Editors are indeed followed by (a cura di)).  Taking a quick look at the styleguide, looks like you'll want the label as `form="short"`in parentheses after the name throughout.